### ISSUE TITLE
Web Console is broken in 6.1 RC1

### ISSUE THREAD

**Issue by gobijan** [rails/rails#40532]
### Steps to reproduce
Generate a new project and call `console` in a view or a controller.

### Expected behavior
Web Console should come up.

### Actual behavior
Exception is thrown: `ArgumentError: wrong number of arguments (given 2, expected 3)`

```
ArgumentError: wrong number of arguments (given 2, expected 3)
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/actionview-6.1.0.rc1/lib/action_view/base.rb:230:in `initialize'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/web-console-4.0.4/lib/web_console/template.rb:20:in `new'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/web-console-4.0.4/lib/web_console/template.rb:20:in `render'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/web-console-4.0.4/lib/web_console/middleware.rb:35:in `block in call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/web-console-4.0.4/lib/web_console/middleware.rb:17:in `catch'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/web-console-4.0.4/lib/web_console/middleware.rb:17:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/actionpack-6.1.0.rc1/lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/railties-6.1.0.rc1/lib/rails/rack/logger.rb:37:in `call_app'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/railties-6.1.0.rc1/lib/rails/rack/logger.rb:26:in `block in call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activesupport-6.1.0.rc1/lib/active_support/tagged_logging.rb:99:in `block in tagged'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activesupport-6.1.0.rc1/lib/active_support/tagged_logging.rb:37:in `tagged'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activesupport-6.1.0.rc1/lib/active_support/tagged_logging.rb:99:in `tagged'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/railties-6.1.0.rc1/lib/rails/rack/logger.rb:26:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/sprockets-rails-3.2.2/lib/sprockets/rails/quiet_assets.rb:13:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/actionpack-6.1.0.rc1/lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/actionpack-6.1.0.rc1/lib/action_dispatch/middleware/request_id.rb:26:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rack-2.2.3/lib/rack/method_override.rb:24:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rack-2.2.3/lib/rack/runtime.rb:22:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/activesupport-6.1.0.rc1/lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/actionpack-6.1.0.rc1/lib/action_dispatch/middleware/executor.rb:14:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/actionpack-6.1.0.rc1/lib/action_dispatch/middleware/static.rb:24:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rack-2.2.3/lib/rack/sendfile.rb:110:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/actionpack-6.1.0.rc1/lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rack-mini-profiler-2.2.0/lib/mini_profiler/profiler.rb:368:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/webpacker-5.2.1/lib/webpacker/dev_server_proxy.rb:25:in `perform_request'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/rack-proxy-0.6.5/lib/rack/proxy.rb:57:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/railties-6.1.0.rc1/lib/rails/engine.rb:535:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/puma-5.0.4/lib/puma/configuration.rb:239:in `call'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/puma-5.0.4/lib/puma/request.rb:76:in `block in handle_request'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/puma-5.0.4/lib/puma/thread_pool.rb:337:in `with_force_shutdown'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/puma-5.0.4/lib/puma/request.rb:75:in `handle_request'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/puma-5.0.4/lib/puma/server.rb:430:in `process_client'
        from /Users/bijan/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/puma-5.0.4/lib/puma/thread_pool.rb:145:in `block in spawn_thread'
```

### System configuration
**Rails version**: 6.1 RC1

**Ruby version**: 2.7.2

There is also an issue open in the web-console repo but I'm not sure wether it should be tracked here or there:
https://github.com/rails/web-console/issues/303

**Comment by stephannv** [comment]
I think this will solve the problem: https://github.com/rails/web-console/pull/304

**Comment by gsamokovarov** [comment]
I'll test `web-console` a bit more against the Rails 6.1 RC and release a version that works with it.

**Comment by gsamokovarov** [comment]
I have released `web-console`, `4.1.0` with @stephannv's fix. We may bump the  `web-console` version in newly generated Rails 6.1 applications.

https://github.com/rails/rails/blob/f61a38a16018551c902cdadfa6325cef5b582970/railties/lib/rails/generators/rails/app/templates/Gemfile.tt#L52

This may be a good first time contribution to Rails as well. @stephannv do you wanna give it a go?

**Comment by stephannv** [comment]
Thank you so much for the opportunity, @gsamokovarov . I will open the pull request with this version bump.
